collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -267,7 +270,10 @@ public void onCreate() {
         new LocaleHelper().updateLocale(this);
         singleton = this;
 
-        PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
+        if (shouldLoadDefaultValues()) {
+            GeneralSharedPreferences.getInstance().loadDefaultValues();
+            AdminSharedPreferences.getInstance().loadDefaultValues();
```

Comment:
Maybe move these to a separate method to keep the onCreate small? Or move the whole check and load to a  `checkAndLoadDefaultValues()` method?